### PR TITLE
Add NodeJS image README

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
This is related to #993.

### CODE REVIEW

File: src/bci_build/package/nodejs/README.md.j2 [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} node script.js
+```
+
+## Additional tools
+
+The following additional tools are included in the image:
+
+- git-core
+
```
Comment (dcermak) [inline src/bci_build/package/nodejs/README.md.j2:53]: I find this section is of questionable usefulness
```suggestion
```

**Review by dcermak — CHANGES_REQUESTED** [review]
Can you also please add a section about how to obtain `yarn`? (you have to install it via `npm install -g yarn`)

**Comment by alexandrevicenzi** [comment]
> Can you also please add a section about how to obtain `yarn`? (you have to install it via `npm install -g yarn`)

I left it out on purpose.

Why yarn, and why not also pnpm or bun?
If I add one, I feel the need to add everything else.
Indeed, yarn is more popular than pnpm or bun, but npm is the default.

In the Python image we don't mention how to use/install Poetry or Conda.

File: src/bci_build/package/nodejs/README.md.j2 (hunk 1 above)
Comment (alexandrevicenzi) [reply src/bci_build/package/nodejs/README.md.j2:53]: Yes, I had a similar feeling.

Comment (dirkmueller) [reply src/bci_build/package/nodejs/README.md.j2:53]: imho we should only document it if we have a test that enforces git-core being available. 

I think generally it is a good idea to list the packages that are core and auxiliary installed but that requires a bit more plumbing in the dockerfile generator to not having this listed manually in the readmes.

Comment (alexandrevicenzi) [reply src/bci_build/package/nodejs/README.md.j2]: I removed this section.

**Review by dcermak — APPROVED** [review]
> Why yarn, and why not also pnpm or bun?

Fair enough, the dockerhub image also doesn't mention it, so let's skip it.

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**